README.md

```diff
@@ -458,6 +460,12 @@ export default function PostList() {
 }
 ```
 
+## File uploads
+
+`graphql-hooks` complies with the [GraphQL multipart request spec](https://github.com/jaydenseric/graphql-multipart-request-spec), allowing files to be used as query or mutation arguments. The same spec is also supported by popular GraphQL servers, including [Apollo Server](https://www.apollographql.com/docs/apollo-server) (see list of supported servers [here](https://github.com/jaydenseric/graphql-multipart-request-spec#server)).
+
+If there are files to upload, the request's body will be a [`FormData`](https://developer.mozilla.org/docs/Web/API/FormData) instance conforming to the GraphQL multipart request spec.
+
```

Comment:
@olistic Do you mind adding a concise code example here - perhaps the code from the tests?